src/main/webapp/content/scss/global.scss

```diff
@@ -938,6 +938,22 @@ Course Info Bar
     box-shadow: 3px 3px 6px $mat-autocomplete-visible-box-shadow;
 }
 
+.mat-select-panel {
+    background: $metis-course-discussion-select-bg;
+    -webkit-box-shadow: 3px 3px 6px $mat-autocomplete-visible-box-shadow;
+    -moz-box-shadow: 3px 3px 6px $mat-autocomplete-visible-box-shadow;
+    box-shadow: 3px 3px 6px $mat-autocomplete-visible-box-shadow;
+    min-width: fit-content;
+    max-height: fit-content !important;
+    position: absolute !important;
```

Comment:
Can you somehow increase the CSS selector specificity instead of using !important here?

Reply:
Sure